developers/weaviate/model-providers/_includes/provider.generative.py

```diff
@@ -215,6 +215,54 @@
 # clean up
 client.collections.delete("DemoCollection")
 
+# START BasicGenerativeFriendliAI
+from weaviate.classes.config import Configure
+
+client.collections.create(
+    "DemoCollection",
+    generative_config=Configure.Generative.friendliai()
+    # Additional parameters not shown
+)
+# END BasicGenerativeFriendliAI
+
+# clean up
+client.collections.delete("DemoCollection")
+
+# START GenerativeFriendliAICustomModel
+from weaviate.classes.config import Configure
+
+client.collections.create(
+    "DemoCollection",
+    # highlight-start
+    generative_config=Configure.Generative.friendliai(
+        model = "meta-llama-3-70b-instruct",
+    )
+    # highlight-end
+    # Additional parameters not shown
+)
+# END GenerativeFriendliAICustomModel
+
+# clean up
+client.collections.delete("DemoCollection")
+
+# START FullGenerativeFriendliAI
+from weaviate.classes.config import Configure
+
+client.collections.create(
+    "DemoCollection",
+    generative_config=Configure.Generative.friendliai(
+        # # These parameters are optional
+        model = "meta-llama-3-70b-instruct",
+        max_tokens = 500,
+        temperature = 0.7,
+        base_url = "https://inference.friendli.ai"
+    )
+)
+# END FullGenerativeFriendliAI
```

Comment:
What does this directive do?

what does this script do?

Reply:
This script just shows the way to use weaviate using our module.
# START ~~ # END ~~ directives are just markdown marker that insert the code block into a .md file.
You can find (startMarker="# START FriendliAIInstantiation", endMarker="# END FriendliAIInstantiation") from generative.md file.

<img width="995" alt="Screenshot 2024-08-21 at 11 20 59 AM" src="https://github.com/user-attachments/assets/7c73c846-270e-49a4-9d25-5be1232e69cd">